src/DataService/DataObjectFactory.php

```diff
@@ -128,6 +128,28 @@ protected function deserialize($data, string $class)
 
         $object = $this->getObjectSerializer($cached)->deserialize($data, $class, 'json');
 
+        // Adds any missing custom field classes. A given object may not
+        // contain any set fields in a given custom field namespace. In that
+        // case, the serializer will never create a field class. This is fine
+        // until calling code wants to check to see if a given field exists.
+        // By ensuring we add in any missing custom field definitions, we
+        // ensure a valid object is returned instead of null.
+        if ($object instanceof ObjectInterface) {
+            $customFields = $object->getCustomFields();
+            $remaining = array_diff_key($this->dataService->getCustomFields(), $customFields);
+
+            /**
+             * @var string
```

Comment:
Should this be `@var string $namespace` ?

Reply:
Yes, it should!

Reply:
Hm. php-cs-fixer is removing that. I'll try a different syntax.